osquery/utils/conversions/windows/windows_time.cpp

```diff
@@ -27,4 +29,38 @@ LONGLONG longIntToUnixtime(LARGE_INTEGER& li) {
   return ull.QuadPart / 10000000ULL - 11644473600ULL;
 }
 
+// Convert little endian Windows FILETIME to unix timestamp
+LONGLONG littleEndianToUnixTime(const std::string& time_data) {
+  // If timestamp is zero dont convert to UNIX Time
+  if (time_data == "0000000000000000") {
+    return 0LL;
+  } else {
```

Comment:
```suggestion
  } 
```
shouldn't need the else with an early return

Reply:
fixed